Dockerfile

```diff
@@ -1,13 +1,6 @@
-FROM registry.access.redhat.com/ubi9/nodejs-18 as builder
+FROM registry.access.redhat.com/ubi9/nodejs-18
```

Comment:
I would have thought this PR would result in a bigger image at the end -- based off a nodejs image rather than plain old nginx -- and not not be much much faster, since the bit you are removing is just copying the compiled files to the nginx image.

`npm ci` and `npm run build --workspace=standalone` are surely the lengthy parts.

If you are worried about the github pipeline times, we could consider reducing the number of archs supported and potentially taking out the build step.